### PR TITLE
Update communication links for the forum

### PR DESCRIPTION
**Pull request by oraNod**
##### SUMMARY

As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR updates communication links to point to the forum and de-emphasize IRC and mailing lists. Similar PRs are being raised across all Ansible ecosystem projects. Thanks!

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

n/a

### CODE REVIEW

**Comment by oraNod** [comment]
@Andersson007 :eyes:

**Comment by bcoca** [comment]
Something I have been considering for a while, just refer everyone to the communications page so we don't have to make any changes in this repo anymore.

**Comment by oraNod** [comment]
> Something I have been considering for a while, just refer everyone to the communications page so we don't have to make any changes in this repo anymore.

Seems fine to me, especially for the templates. Could we keep the changes in the README here so we provide direct links to forum tags?

**Review by Andersson007 — APPROVED** [review]

**Comment by oraNod** [comment]
@bcoca PTAL. I also noticed the doc urls are inconsistent. It's a pretty minor thing but I feel like they should point to `devel` instead of `latest` so users get the "freshest" set of docs. Thought I'd ask before making any changes because it touches several urls in the readme, etc. Thanks.

**Review by Andersson007 — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,11 +4,14 @@ Hi! Nice to see you here!
...
 ## QUESTIONS ?
 
-Please see the [Community Guide](https://docs.ansible.com/ansible/latest/community/index.html) for information on
-how to ask questions on the [mailing lists](https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information) and IRC.
+If you have questions about anything related to Ansible, get in touch with us!
+See [Communicating with the Ansible community](https://docs.ansible.com/ansible/devel/community/communication.html) to find out how.
 
-The GitHub issue tracker is not the best place for questions for various reasons,
-but both IRC and the mailing list are very helpful places for those things, as the community page explains best.
+The [Community Guide](https://docs.ansible.com/ansible/latest/community/index.html) also explains how to contribute
```
Comment (gundalow) [inline .github/CONTRIBUTING.md]: This points to `latest`, where as we are using `devel` for communication

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ contact_links:
 - name: 📝 Ansible Code of Conduct
   url: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html?utm_medium=github&utm_source=issue_template_chooser
   about: ❤ Be nice to other members of the community. ☮ Behave.
-- name: 💬 Talks to the community
+- name: 💬 Talk to the community
   url: https://docs.ansible.com/ansible/latest/community/communication.html?utm_medium=github&utm_source=issue_template_chooser#mailing-list-information
```
Comment (gundalow) [inline .github/ISSUE_TEMPLATE/config.yml]: devel?

File: README.md [hunk 3]
```diff
@@ -40,21 +40,33 @@ features and fixes, directly. Although it is reasonably stable, you are more lik
...
+* [Social Spaces](https://forum.ansible.com/c/chat/4): Meet and interact with fellow enthusiasts.
+* [News & Announcements](https://forum.ansible.com/c/news/5): Track project-wide announcements including social events.
+* [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn): Get release announcements and important changes.
+
+For more ways to get in touch, see [Communicating with the Ansible community](https://docs.ansible.com/ansible/devel/community/communication.html).
+
+## Contribute to Ansible
+
+* Check out the [Contributor's Guide](./.github/CONTRIBUTING.md).
 * Read [Community Information](https://docs.ansible.com/ansible/latest/community) for all
```
Comment (gundalow) [inline README.md]: devel?

**Review by gundalow — COMMENTED** [review]
Let's decide if we want to link to `devel` or `latest` and do a bulk update to these files.

**Comment by oraNod** [comment]
> Let's decide if we want to link to `devel` or `latest` and do a bulk update to these files.

My 2c is that for the comms details, we should point to `devel` because we regularly update that content. The comms links should be versionless like the README. For code bits `latest` is best because things are tested but that doesn't really apply to comms.

I suppose one concern might be directing users to `devel` might be risky in the sense that they might navigate to other parts of the doc instead of staying on `latest`. Not sure how much of a concern that is though.

**Comment by Andersson007** [comment]
> > Let's decide if we want to link to `devel` or `latest` and do a bulk update to these files.
> 
> My 2c is that for the comms details, we should point to `devel` because we regularly update that content. The comms links should be versionless like the README. For code bits `latest` is best because things are tested but that doesn't really apply to comms.
> 
> I suppose one concern might be directing users to `devel` might be risky in the sense that they might navigate to other parts of the doc instead of staying on `latest`. Not sure how much of a concern that is though.

i support it

**Review by gundalow — APPROVED** [review]

**Pull request merged**